### ISSUE TITLE
[Designer] Bug 31569277: [Programmatic access - Adaptive Cards - Activity Update.Json]: An element's IsOffScreen property must be true when it's clickable point is off screen.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7637]
### Problem Description

[Bug 31569277](https://microsoft.visualstudio.com/OS/_workitems/edit/31569277): [Programmatic access - Adaptive Cards - Activity Update.Json]: An element's IsOffScreen property must be true when it's clickable point is off screen.

User Experience:

It will not impact the screen reader users but this will be good code practice. 

Test Environment:

Application: Adaptive Cards

OS version: 2004 (OS Build 21301.1000)

Application Version: 2101.28.1.0

Tool Used: Accessibility Insights 

Pre-Requisite: Launch the Adaptive Card application.

Repro steps:

1. Launch the Adaptive Card application.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to ActivityUpdate.Json button and activate it.

4. Run K4D on left pane and verify the issue.

Actual Result:

Value of IsOffScreen property is defined False.

Expected result:

Value of IsOffScreen property should be defined True.

Observation: This issue is also repro on the text "View" for the button. PFA attachment.

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/3f5a9542-a524-438d-bdc9-8f19a28fda33)
